### PR TITLE
Welsh language - persist on error page

### PR DESCRIPTION
**Pull request by ScottDormand96**
https://eaflood.atlassian.net/browse/IWTF-2917

Welsh language persist when an erro page loads up

### CODE REVIEW

**Review by MickStein — APPROVED** [review]

File: packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js [hunk 1]
```diff
@@ -116,3 +122,67 @@ describe('The user', () => {
...
+    state: () => {},
+    redirect: jest.fn(() => ({
+      takeover
+    }))
+  })
+  const initialisedSessionManager = sessionManager(sessionCookieName)
+
+  beforeEach(jest.clearAllMocks)
+
+  it("redirects to agreed handler when status is already agreed and request path isn't in exempt set", async () => {
```
Comment (irisfaraway) [inline packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js:155]:
```suggestion
  it('redirects to agreed handler when status is already agreed and request path isn't in exempt set', async () => {
```

I know I'm nitpicking here but :)

File: packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js [hunk 2]
```diff
@@ -116,3 +122,67 @@ describe('The user', () => {
...
   })
 })
+
+describe('takeover by agreed handler once agreed flag is set', () => {
+  const sessionCookieName = 'garibaldi'
+  const getMockRequest = () => ({
+    path: 'crazy-paving',
+    state: {
+      [sessionCookieName]: {
+        id: 'a disappointing currant embedded biscuit'
```
Comment (irisfaraway) [inline packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js:132]: 😂

**Review by irisfaraway — CHANGES_REQUESTED** [review]
One very minor style thing, other than that LGTM

File: packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js (hunk 1 above)
Comment (jaucourt) [reply packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js:155]: I would normally use single quotes, but note the apostrophe in isn't

Comment (irisfaraway) [reply packages/gafl-webapp-service/src/session-cache/__tests__/session-manager.spec.js:155]: Oops. You're totally right!

**Review by irisfaraway — APPROVED** [review]

**Pull request merged**